ctapipe/core/tool.py

```diff
@@ -1,21 +1,41 @@
 """Classes to handle configurable command-line user interfaces."""
 import logging
 import logging.config
-import textwrap
-from abc import abstractmethod
-import pathlib
 import os
+import pathlib
 import re
+import textwrap
+from abc import abstractmethod
+from typing import Union
+
+try:
+    import yaml
+
+    HAS_YAML = True
+except ImportError:
+    HAS_YAML = False
+    pass  # no support for YAML
+
+try:
+    import tomli
```

Comment:
`tomli as toml` or `from tomli import load as load_toml`? Otherwise the code below won't run